features/other_a/admin_tasks.feature

```diff
@@ -342,24 +342,126 @@ Feature: Admin tasks
   When I edit known issues
   Then I should see "Known issue was successfully updated"
 
-  Scenario: Add a locale
+<<<<<<< HEAD
```

Comment:
All of this account creation test stuff was moved to different files, so lines 345-443 should be removed.